example_test.go

```diff
@@ -284,3 +284,26 @@ func ExampleEmulate() {
 		panic(err)
 	}
 }
+
+func ExamplePrintToPDF() {
+	ctx, cancel := chromedp.NewContext(context.Background(), chromedp.WithDebugf(log.Printf))
+	defer cancel()
+
+	var buf []byte
+	if err := chromedp.Run(ctx,
+		chromedp.Navigate(`https://godoc.org/github.com/chromedp/chromedp`),
+		chromedp.ActionFunc(func(ctx context.Context) error {
+			var err error
+			buf, _, err = page.PrintToPDF().
+				WithDisplayHeaderFooter(false).
+				WithLandscape(true).
+				Do(ctx)
+			return err
+		})); err != nil {
```

Comment:
I prefer to put the call-closing `)` on a separate line, so that all call arguments are consistently on their own lines. That also makes the indentation a bit nicer to follow, I think.